### PR TITLE
Kehyde/fix profileview

### PR DESCRIPTION
**Pull request by futurethang**

### CODE REVIEW

File: ProfileView/package.json [hunk 1]
```diff
@@ -27,21 +27,21 @@
...
   "peerDependencies": {
     "@cjaas/common-components": "2.2.0",
-    "@momentum-ui/core": "^19.15.3",
-    "@momentum-ui/icons": "^8.20.1",
-    "@momentum-ui/utils": "^6.2.15",
-    "@momentum-ui/web-components": "^2.4.0",
+    "@momentum-ui/core": "^19.10.0",
+    "@momentum-ui/icons": "^8.8.2",
+    "@momentum-ui/utils": "^6.2.12",
+    "@momentum-ui/web-components": "2.4.0",
```
Comment (elenagiere) [inline ProfileView/package.json:33]: should this be with a caret?

Comment (venkateshwarv) [reply ProfileView/package.json:33]: Its best to get rid of the caret

Comment (futurethang) [reply ProfileView/package.json:33]: @venkateshwarv in all cases seen here?

**Comment by futurethang** [comment]
Superseded by #23

**Pull request closed without merging**